nengo_spinnaker/builder/model.py

```diff
@@ -49,24 +51,11 @@ def add_connection(self, source_object, source_port, signal_parameters,
             treated.
         """
         # Combine the signal parameters with the transmission parameters
-        signal_parameters = (signal_parameters, transmission_parameters)
+        pars = (signal_parameters, transmission_parameters)
 
         # See if we can combine the connection with an existing set of
         # transmission parameters.
-        pars_list = self._connections[source_object][source_port]
-        for pars, sinks in pars_list:
-            # If the transmission parameters are equivalent then we can add the
-            # sink parameters to the list of sinks.
-            if pars == signal_parameters:
-                break
-        else:
-            # If we can't combine with an existing connection then we need to
-            # add a new transmission parameters and set of sinks.
-            psp = _ParsSinksPair(signal_parameters)
-            pars_list.append(psp)
-            sinks = psp.sinks
-
-        # Add the parameters to the list of sinks
+        sinks = self._connections[source_object][source_port][pars]
```

Comment:
Maybe add a note that the final ``pars`` is a hashable tuple of parameters (or something)